src/components/Sentence/index.jsx

```diff
@@ -1,9 +1,24 @@
-import React from 'react';
+import React, { useState, useEffect } from "react";
 
 export default function Sentence({ word, sentence }) {
+  const [wordTimer, setWordTimer] = useState(false);
+
+  useEffect(() => {
+    setWordTimer(false);
+    const timer = setTimeout(() => {
+      setWordTimer(true);
+    }, 1500);
+    return () => clearTimeout(timer);
+  }, [word]);
+
   return (
     <>
-      <h2>{word}</h2>
+      <h2
+        style={{ visibility: wordTimer ? "hidden" : "" }}
+        className={wordTimer ? "hidden" : ""}
```

Comment:
Do we need both line 17 and 18. It looks like they will eventually do the same thing?

Reply:
They do do the same thing. I left it to play around with actually hiding it, but will deal with it during the styling process